OpenRA.Mods.Common/Traits/World/Locomotor.cs

```diff
@@ -484,7 +487,7 @@ void UpdateCellBlocking(CPos cell)
 
 					var crushables = actor.TraitsImplementing<ICrushable>();
 					var mobile = actor.OccupiesSpace as Mobile;
-					var isMovable = mobile != null;
+					var isMovable = mobile != null && !mobile.IsTraitDisabled && !mobile.IsTraitPaused && !mobile.RequireForceMove;
```

Comment:
Think we need to add  something similar to what we going for in #17159 to the IsTraitDisabled  and IsTraitPaused

Reply:
Done.